oem/zt/zt.go

```diff
@@ -0,0 +1,96 @@
+package zt
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stmcginnis/gofish/common"
+	"github.com/stmcginnis/gofish/redfish"
+	"io"
+	"net/http"
+)
+
+var (
+	submitTestEventTarget = "/redfish/v1/EventService/Actions/EventService.SendTestEvent"
+)
+
+type (
+	subscriptionPayloadZT struct {
+		Destination string                           `json:"Destination"`
+		HTTPHeaders map[string]string                `json:"HttpHeaders,omitempty"`
+		Protocol    redfish.EventDestinationProtocol `json:"Protocol,omitempty"`
+		Context     string                           `json:"Context,omitempty"`
+	}
+
+	// subscribeResponseType zt uses a unique subscription response.
+	subscribeResponseType struct {
+		OdataContext        string `json:"@odata.context"`
+		OdataEtag           string `json:"@odata.etag"`
+		OdataID             string `json:"@odata.id"`
+		OdataType           string `json:"@odata.type"`
+		Context             string `json:"Context"`
+		DeliveryRetryPolicy string `json:"DeliveryRetryPolicy"`
+		Description         string `json:"Description"`
+		Destination         string `json:"Destination"`
+		EventFormatType     string `json:"EventFormatType"`
+		ID                  int    `json:"ID"`
+		Name                string `json:"Name"`
+		Protocol            string `json:"Protocol"`
+		Status              struct {
+			Health       string `json:"Health"`
+			HealthRollup string `json:"HealthRollup"`
+			State        string `json:"State"`
+		}
```

Comment:
This could just use the `Status` struct: https://github.com/stmcginnis/gofish/blob/main/common/types.go#L261

Reply:
done